webapp-django/crashstats/tokens/utils.py

```diff
@@ -0,0 +1,7 @@
+from raven.processors import SanitizePasswordsProcessor
+
+
+class RavenSanitizeAuthTokenProcessor(SanitizePasswordsProcessor):
+    """Extend the standard SanitizePasswordsProcessor with one more key
+    which is 'Auth-Token' which is what we use in our API."""
+    FIELDS = tuple(SanitizePasswordsProcessor.FIELDS) + ('auth-token',)
```

Comment:
`FIELDS` is a frozenset:

https://github.com/getsentry/raven-python/blob/908f3711b175c8184244ee9d648d625e446007a9/raven/processors.py#L75

I think I'd probably make sure it stays a frozenset.

Relatedly, it's unenthusing that it's this difficult to add other things to the sanitization list with raven.